src/work-allocation-2/containers/task-list/task-list.component.html

```diff
@@ -1,101 +1,98 @@
-        <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
-                (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
-      </th>
-      <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
-        <exui-task-field [task]="rowData" [config]="field"></exui-task-field>
-      </td>
-    </ng-container>
+  <!-- Column Headers and Cells -->
+<ng-container *ngIf="tasksTotal">
+  <ng-container *ngFor="let field of fields | paginate: { itemsPerPage: pagination?.page_size, currentPage: pagination?.page_number, totalItems: tasksTotal }" [cdkColumnDef]="field.name">
+    <th class="govuk-table__header" [id]="'header_' + field.sortName" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
+      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
+    </th>
+    <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
+      <exui-work-field [workField]="rowData" [config]="field"></exui-work-field>
+    </td>
   </ng-container>
-  <!-- Below container sets up initial task list headers when not yet populated as session storage of page number causing issues with above container -->
-  <ng-container *ngIf="!tasksTotal">
-    <ng-container *ngFor="let field of fields" [cdkColumnDef]="field.name">
-      <th class="govuk-table__header" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
-        <button class="column-sort" *ngIf="field.columnLabel && !field.disableSort" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
-                (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
-        <h1 class="column-label" *ngIf="field.disableSort">{{field.columnLabel}}</h1>
-      </th>
-      <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
-        <exui-task-field [task]="rowData" [config]="field"></exui-task-field>
-      </td>
-    </ng-container>
+</ng-container>
+<!-- Below container sets up initial task list headers when not yet populated as session storage of page number causing issues with above container -->
+<ng-container *ngIf="!tasksTotal">
+  <ng-container *ngFor="let field of fields | paginate: { itemsPerPage: pagination?.page_size, currentPage: pagination?.page_number, totalItems: tasksTotal }" [cdkColumnDef]="field.name">
+    <th class="govuk-table__header" [id]="'header_' + field.sortName" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
+      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
```

Comment:
_Nitpick:_
Another extraneous space before the `>`.

Reply:
Formatting the document appears to have resolved this.